### DISCUSSION TITLE
Feature Request: Filter for unarchived repos

### DISCUSSION THREAD

**Discussion by ChaimStanton** [community/community#7619] [Other Feature Feedback, Questions, & Ideas]
When you click on your profile image and go to repositories, there is an option to filter for only archived repositories next to the search bar. However, there is no option to filter for only unarchived repos.
![filter options including archive but excluding archive](https://user-images.githubusercontent.com/38401049/141408824-84cbe380-986e-4a6b-a66c-e2f7c7f1906a.png)

A potential workaround for this is to use `archived:true` or `archived:false` as per @clarkbw's suggestion [here](https://github.com/isaacs/github/issues/1363), however [that repo and discussions have now depreaciated](https://github.com/isaacs/github/blob/f6fc600746d6fba5056b8b53e4d28b0141b78a1b/README.md).

This is similar to #6906 but that only refers to an organisation repo, not on the personal profile.

**Comment by jeapostrophe** [comment]
Yes, I really want this too.
